### PR TITLE
feat: custom matcher toContainText

### PR DESCRIPTION
**Pull request by cexbrayat**
Usage: `expect(page.title).toContainText('Welcome');`

On top of #1

### CODE REVIEW

File: projects/ngx-speculoos/src/lib/matchers.ts [hunk 1]
```diff
@@ -60,6 +60,38 @@ const speculoosMatchers: jasmine.CustomMatcherFactories = {
...
+        return { pass: false, message: `Expected to check textContent '${expected}' on element, but element was not a TestElement` };
+      }
+      const actual = el.textContent;
+      if (!actual) {
+        return {
+          pass: isNegative,
+          message: `Expected element to ${isNegative ? 'not ' : ''}contain textContent '${expected}', but had no textContent`
+        };
+      }
+      const pass = actual.indexOf(expected) !== -1;
```
Comment (jnizet) [inline projects/ngx-speculoos/src/lib/matchers.ts]: actual.includes(expected)?

Comment (cexbrayat) [reply projects/ngx-speculoos/src/lib/matchers.ts]: This is es2016, so I had to change the tsconfig.json to add `es2016` if I wanted to use and I didn't know if that was good or bad...

**Comment by jnizet** [comment]
Shouldn't we have a 'toHaveText' matcher, that would test equality? It would be useful if strictness is needed, or simply to test that en element has an empty text content.

**Comment by cexbrayat** [comment]
Yeah I hesitated. I can easily add it, PR incoming.

File: projects/ngx-speculoos/src/lib/matchers.ts (hunk 1 above)
Comment (jnizet) [reply projects/ngx-speculoos/src/lib/matchers.ts]: I don't measure all the implications, so leave it like this.

**Comment by cexbrayat** [comment]
Rebased, tell me if that's OK to merge

**Comment by jnizet** [comment]
LGTM

**Pull request merged**